1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,9 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter(
+  (value) => value instanceof Array && value.length > 1
```

Comment:
I think `value.length === 2` is better than `value.length > 1`. Your solution would fail if a test contained an array with 3 items.